doc/releases/changelog-0.37.0.md

```diff
@@ -6,57 +6,171 @@
 
 <h4>Execute faster with Default Tensor 🔗</h4>
```

Comment:
```suggestion
<h4>Execute bigger with Default Tensor 🔗</h4>
```

I would suggest this reframing since the main point of the device is size and not speed. In fact, most <20q circuits are running slower on default.tensor than default.qubit, as the point is more scalability (at this point at least)

Reply:
Fair point! Wasn't aware of DT being slower than DQ for <20 qubits 🤔... Is this strictly implementation that's causing this? I would expect that I could at least get super fast results with a crappy bond dimension for upwards of ~12 qubits, no?

Reply:
Yeah, the devil is in the detail: So for "typical" circuits you have to go to larger qubit sizes to get a chance to beat default.qubit, even for small bond dimensions. There are some known performance bottlenecks from re-canonicalizing the MPS state that we havent addressed yet, plus overall there are some competing overheads from SVDs that make it slower.

Check this example with 8 qubits. Even with trivial bond dimension of 2 DQ is faster:
```
dataset = qml.data.load("qchem", molname="H4", bondlength=0.92, basis="STO-3G")[0]

H = dataset.hamiltonian
n_wires = len(H.wires)

def circuit():
    qml.BasisState(dataset.hf_state, wires=H.wires)
    for op in dataset.vqe_gates:
        qml.apply(op)
    return qml.expval(H)

bond_dims = 2**np.arange(2, (n_wires//2)+1) # maximum required bond dinemsion is 2**(n_wires//2) = 64
ress = []

bond_dim = 2
mps = qml.device("default.tensor", wires=n_wires, method="mps", max_bond_dim=bond_dim, contract="nonlocal")
dev = qml.device("default.qubit")
qnode_mps = qml.QNode(circuit, mps)
qnode_def = qml.QNode(circuit, dev)

%timeit res = qnode_mps()
179 ms ± 5.34 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
%timeit res = qnode_def()
116 ms ± 3.49 ms per loop (mean ± std. dev. of 7 runs, 10 loops each)
```

This is one example of how performance can look like (here the circuit depth is constant)
![image](https://github.com/PennyLaneAI/pennylane/assets/43949391/89cea045-7963-439d-9c91-f9e61df2cb33)
The take away is that typically the break-even point is somewhere between 12-20 qubits, depending on the circuit and bond dimension. So people switching out the device for small examples will likely first see a performance regression (before they scale up qubit count)

Reply:
```suggestion
<h4>Execute wide circuits with Default Tensor 🔗</h4>
```